src/site/views/event/tmpl/default.php

```diff
@@ -101,7 +102,73 @@
 							</tr>
 							</tfoot>
 						</table>
+						<table class="table table-hover">
+							<thead>
+							<tr>
+								<th>University</th>
+								<th>Tickets Sold</th>
+							</tr>
+							</thead>
+							<tbody>
+							<?php
+							$universityCounts = array();
+
+							foreach ($eventAttendees as $person)
+							{
+								$universityCounts[$person['Uni']] += 1;
+							}
+							arsort($universityCounts);
+							foreach ($universityCounts as $uni => $count)
+							{
+								echo "<tr>\n";
+								echo "<td>{$uni}</td>";
+								echo "<td>{$count}</td>";
+								echo "</tr>\n";
+							}
+							?>
+							</tbody>
+						</table>
+
+						<?php
+						  // Only want to show the full attendees table when the club committees actually need it
+						  $registrationClosed = time() > (24 * 60 * 60 + strtotime($item->event_date_close));
+							$eventFinished = time() > (72 * 60 * 60 + strtotime($item->event_date));
```

Comment:
Interesting indentation here.

Reply:
unsure how that happened, sorry

Reply:
indentation looks fine on my editor, hmm